superset/assets/javascripts/explore/propTypes/legacyMetricType.js

```diff
@@ -0,0 +1,6 @@
+import PropTypes from 'prop-types';
```

Comment:
I don't see this being used anywhere, what's this for?

Reply:
Oh, good catch! I actually don't use this file anywhere at all! I will delete it.